.travis.yml

```diff
@@ -21,14 +21,15 @@ before_install:
   - cd $HOME/gopath/src/${PROJ}
 
 install:
-  - go get -v -t -d
+  - go get -v -t -d ./...
+  - go get github.com/elastic/beats/vendor/github.com/pierrre/gotestcover
```

Comment:
Nice. That is an interesting benefit of having the vendored packages checked in ...